Backend/CarRentalApp/CarRentalApp/Services/Identity/UserService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using CarRentalApp.Models.Data;
+using CarRentalApp.Models.DTOs.Requests;
+using CarRentalApp.Services.Data;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace CarRentalApp.Services.Identity
+{
+    public class UserService : IUserService
+    {
+        private IUserRepository _userRepository;
+
+        public UserService(IUserRepository userRepository)
+        {
+            _userRepository = userRepository;
+        }
+
+        public async Task<User?> GetValidUserAsync(UserLoginDTO user2Login)
```

Comment:
What is the definition of "valid user"? Is it the responsibility of user service to check user passwords?

Reply:
> Is it the responsibility of user service to check user passwords?
I decided to unite all operations with users in this service